packages/dev/core/src/Cameras/arcRotateCamera.ts

```diff
@@ -1129,6 +1137,9 @@ export class ArcRotateCamera extends TargetCamera {
             this.radius = this.upperRadiusLimit;
             this.inertialRadiusOffset = 0;
         }
+        if (this.lowerTargetYLimit !== null && this.target.y < this.lowerTargetYLimit) {
+            this.target.y = this.lowerTargetYLimit;

```

Comment:
is it basically not a this.target.y = Math.max(this.target.y, this.lowerTargetYLimit) ?

Reply:
This was my initial though as well, but the TS compiler blames because `null` is not a valid input for `Math.max`.
I guess that's why are `lowerAlphaLimit` and `upperAlphaLimit` are solved with this `if` pattern as well.

Reply:
Maybe we should initialize at -infinity so we prevent different objects maps and simplify the code ?

Reply:
Sounds good to me, as it would solve the inspector issue as well.
Should I also change `lowerAlphaLimit` and `upperAlphaLimit` to `-Infinity` / `Infinity`?

Reply:
looks like an inmprovment ?